### PR TITLE
Feature/readme

### PR DESCRIPTION
**Pull request by kosiakkatrina**
**What**  
Add a readme

**Why**  
To assist any future developers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,17 +2,101 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
 
 ## Getting Started
 
-First, run the development server:
+1. Install dependencies:
+  ```bash
+  yarn
```
Comment (mmmmillar) [inline README.md:7]: with the yarn/npm stuff i think we just put yarn and if they wanna use npm they can? make it succinct

File: README.md [hunk 2]
```diff
@@ -2,17 +2,101 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
...
+
+2. Create a .env file, based on the .env.sample file:
+  ```bash
+  touch .env # then go fill it in!
+  ```
+
+3. Set up DynamoDB local:
+  ```bash
+  brew cask install java # if you don't already have Java
+  npm run dynamo-install
```
Comment (mmmmillar) [inline README.md:20]: shall we just say it requires java? not everyone will use homebrew

File: README.md [hunk 3]
```diff
@@ -2,17 +2,101 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
...
+
+Plan summary
+```bash
+/plans/[id]
+```
+
+Sharing page of the plan
+```bash
+/plan/[id]/share
+```
```
Comment (mmmmillar) [inline README.md:99]: I think we should have an api section too

File: README.md [hunk 4]
```diff
@@ -2,17 +2,134 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
...
+2. Create a .env file, based on the .env.sample file:
+  ```bash
+  touch .env # then go fill it in!
+  ```
+
+3. Set up DynamoDB local:
+  Install Java if you don't already have it
+  Install dynamodb:
+  ```bash
+  sls dynamodb install
```
Comment (mmmmillar) [inline README.md:19]: I don't think this is meant to be sls?

File: README.md [hunk 5]
```diff
@@ -2,17 +2,134 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
...
+
+```bash
+yarn unit-test
+```
+
+## Integration tests
+
+Set these env vars in .env file:
+
+```bash
```
Comment (mmmmillar) [inline README.md:46]: Not bash?

File: README.md [hunk 6]
```diff
@@ -2,17 +2,134 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
...
+NEXT_PUBLIC_API_URL=http://localhost:3000/api
+SMS_API_URL=http://localhost:8080
+```
+
+#### Run cypress integration tests headlessly:
+1. Set these env vars to the same values as your local aws cli config:
+  ```bash
```
Comment (mmmmillar) [inline README.md:53]: ..

File: README.md [hunk 7]
```diff
@@ -2,17 +2,134 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
...
+  ```
+
+## Pages
+Home page
+```bash
+/
+```
+
+Plan summary
+```bash
```
Comment (mmmmillar) [inline README.md:82]: ..

File: README.md [hunk 8]
```diff
@@ -2,17 +2,134 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
...
+  ```
+
+2. Open the test runner:
+  ```bash
+  yarn cypress-open
+  ```
+
+## Pages
+Home page
+```bash
```
Comment (mmmmillar) [inline README.md:77]: ..

File: README.md (hunk 4 above)
Comment (kosiakkatrina) [reply README.md:19]: Hm, I think that's how you can install local DynamoDB. That's the command in the scripts of one of the other projects

**Pull request merged**